Drivers/MotorChannel/Inc/ZP_D_PWMChannel.hpp

```diff
@@ -0,0 +1,33 @@
+#ifndef ZP_D_PWM_CHANNEL_HPP_
+#define ZP_D_PWM_CHANNEL_HPP_
+
+#include "main.h"
+
+#include "ZP_D_MotorChannel.hpp"
+
+class PWMChannel : public MotorChannel {
+    public:
+        PWMChannel(uint16_t pin_num, GPIO_TypeDef* port, TIM_HandleTypeDef* timer,
+					uint16_t timer_channel,uint32_t clock_frequency);
+
+        void setup(void);
+        void set(uint8_t percent);
+    private:
+        uint16_t pin_num_;
+        GPIO_TypeDef* port_;
```

Comment:
I'm pretty sure `pin_num_` and `port_` aren't actually used?

Reply:
Yes, I'll remove these. I left them in because they were there from the LOS driver, and I didn't want to change the syntax